packages/paste-website/src/pages/components/keyboard-key/index.mdx

```diff
@@ -0,0 +1,413 @@
+  <Callout variant="neutral">
+    <CalloutHeading>Are you considering implementing a new keyboard shortcut?</CalloutHeading>
+    <CalloutText>
+      Reach out to us in a{" "}
+      <Anchor
+        href="https://github.com/twilio-labs/paste/discussions"
+        showExternal={true}
+        i18nExternalLinkLabel="Navigates to Paste Github Discussions"
+      >
+        Github Discussion
+      </Anchor>{" "}
+      so we can keep keyboard shortcuts standardized across our platforms by connecting you with other teams, like Flex
+      and Studio, who have already built them into features.
+    </CalloutText>
+  </Callout>
+</Box>
+
+### Accessibility
+
+To make sure users easily understand keyboard commands, use text instead of symbols to write out modifier keys like Command, Control, and Option. Example: Use “Command” instead of “⌘”. Use the abbreviation, e.g., “Cmd”, only when space is limited.
+
+To expose the presence of shortcuts to assistive technologies, use the [`aria-keyshortcuts`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-keyshortcuts) attribute on the element that gets activated by the shortcut.
+
+## Examples
+
+### Default Keyboard Key
+
+Use Keyboard Key for a single key command. Use Keyboard Key Group for key combination commands or shortcuts. When showing a key combination, do not put a “+” between Keyboard Keys or in Keyboard Key Group.
+
+<LivePreview scope={{ KeyboardKeyGroup, KeyboardKey, Paragraph }} language="jsx">
+  {defaultExample}
+</LivePreview>
+
+### Disabled Keyboard Key
+
+Disabled Keyboard Keys should be used when the element a shortcut activates is disabled, like a disabled Menu item or Button
+
+<LivePreview
+  scope={{ KeyboardKeyGroup, KeyboardKey, Paragraph, useMenuState, Menu, MenuButton, MenuItem, ChevronDownIcon, Box }}
+  language="jsx"
+  noInline
+>
+  {disabledExample}
+</LivePreview>
+
+### Pressed Keyboard Key
+
+Use pressed Keyboard Keys to give visual feedback when a key is pressed.
+
+This is especially helpful when onboarding users to keyboard shortcuts. However, use the pressed state thoughtfully, only when it enhances the user experience. It can be distracting in cases where a user is using the keyboard for other interactions, like on a page with form fields.
+
+The `KeyboardKeyGroup` accepts state returned from the [`useKeyCombination`](#useKeyCombination) hook that allows pressedStyling to be enabled. You must also specify `keyEvent` on the component to detect the correct key pressed.
+
+A mapping of key events can be found [here](https://www.freecodecamp.org/news/javascript-keycode-list-keypress-event-key-codes/)
+
+<LivePreview scope={{ KeyboardKeyGroup, KeyboardKey, useKeyCombination, Paragraph }} noInline language="jsx">
+  {pressedExample}
+</LivePreview>
+
+### Keyboard Key Tooltip
+
+The Keyboard Key Tooltip provides a way to show the keyboard shortcut associated with an element when the element is focused via the keyboard or on mouse hover.
```

Comment:
```suggestion
### Keyboard Key within Tooltip

Add Keyboard Key(s) to a [Tooltip](/components/tooltip) to show a shortcut associated with an interactive component. Use the `actionHeader` and `keyCombinationsActions` to display a heading and the associated keyboard shortcut (or shortcuts, if displaying for multiple operating systems/keyboards).
```

Reply:
suggestion: should we add a short sentence about when we recommend using a tooltip? we prob don't want everyone using them in every possible instance